iroh-net/src/endpoint.rs

```diff
@@ -109,12 +113,14 @@ impl Builder {
 
     /// Binds the magic endpoint on the specified socket address.
     ///
-    /// The *bind_port* is the port that should be bound locally.
-    /// The port will be used to bind an IPv4 and, if supported, and IPv6 socket.
-    /// You can pass `0` to let the operating system choose a free port for you.
+    /// The *addr_v4* is the address that should be bound locally on IPv4.
     ///
-    /// NOTE: This will be improved soon to add support for binding on specific addresses.
-    pub async fn bind(self, bind_port: u16) -> Result<Endpoint> {
+    /// You can pass `None` to let the operating system choose a free port for you, and bind to `0.0.0.0`.
+    ///
+    /// The *addr_v6* is the address that should be bound locally on IPv6.
+    ///
+    /// You can pass `None` to let the operating system choose a free port for you, and bind to `[::]`.
```

Comment:
Can't this whole block now be `Binds the magic endpoint.`?  All the details are from an earlier version still?

Reply:
yes, removed